Sources/WinAppDriver/Win32ProcessTree.swift

```diff
@@ -64,23 +79,53 @@ internal class Win32ProcessTree {
         return jobHandle
     }
 
-    private static func createProcessInJob(commandLine: String, jobHandle: HANDLE) throws -> HANDLE {
+    private static func createProcessInJob(
+        commandLine: String,
+        jobHandle: HANDLE,
+        options: ProcessLaunchOptions = ProcessLaunchOptions()
+    ) throws -> HANDLE {
         try commandLine.withCString(encodedAs: UTF16.self) { commandLine throws in
             var startupInfo = STARTUPINFOW()
             startupInfo.cb = DWORD(MemoryLayout<STARTUPINFOW>.size)
+            var redirectStdHandle = false
+
+            let creationFlags =
+                DWORD(CREATE_SUSPENDED) | DWORD(CREATE_NEW_PROCESS_GROUP)
+                | (options.spawnNewConsole ? DWORD(CREATE_NEW_CONSOLE) : 0)
+            if let stdoutHandle = options.stdoutHandle {
+                startupInfo.hStdOutput = stdoutHandle
```

Comment:
We should not set these handles to `INVALID_HANDLE_VALUE` because `INVALID_HANDLE_VALUE` (= -1 = 0xfff...) is not actually invalid but is the current process handle. So here, we use the current process input/output handles as a fallback, which makes the `redirectStdHandle` logic more complex than it should be.

`GetStdHandle()` will return `NULL` if the standard input/output devices have been closed, which is also a valid value to set the `stdoutHandle` option to if you don't want the child process to write anything, for instance. IIUC `NULL HANDLE` (=0) is a different value from Swift `nil`, since the `stdoutHandle` in the `ProcessLaunchOptions` struct is an optional.